sys/usb/usbus/cdc/acm/Kconfig

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_USBUS_CDC_ACM
+    bool "Configure USBUS CDC ACM"
+    depends on MODULE_USBUS_CDC_ACM
+    help
+        Configure the USBUS CDC ACM module via Kconfig.
+
+if KCONFIG_MODULE_USBUS_CDC_ACM
+
+config USBUS_CDC_ACM_STDIO_BUF_SIZE
+    int "Buffer size for STDIN and STDOUT data"
+    default 128
```

Comment:
I just remembered that this is using isrpipe underneath and thus **must** be a power of two. Do you think it makes sense to add that to the help here? It is not a limited set of values so it doesn't have to be a choice type

Reply:
Yes, that should be part of the help string. A safer way would be to model this as the exponent